admin/installer/setup_client.sh

```diff
@@ -53,3 +54,6 @@ ${DOCKER_CERT_HOME}/createclient.exp
 
 # Copy Docker Certificates directory to Ubuntu's home directory.
 cp -r ${DOCKER_CERT_HOME} ${UBUNTU_HOME}
+
+# Signal successful stack creation.
+curl ${SUCCESS_SIGNAL_URL}
```

Comment:
No action required: Just documenting review process here.

Richard downloaded all of the logs from `s3://check.clusterhq.com/logs/` and grepped through all of the logs for `/cloudformation-complete.txt` and found some hits. They seemed to be around yesterday midday which seems to be approximately when Richard and Madhuri were adding this code.

This confirms that we have logging enabled on the bucket and we believe it is logging requests that came from runs of the cloudformation template.

This bucket was not being tracked by qloudstat though... I re-configured qloudstat to also consume the logs from this bucket. I expect that to work, and it's more important that we get this checked in so that we log the installs.